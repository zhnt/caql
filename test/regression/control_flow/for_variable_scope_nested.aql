// Nested loops: outer variable as inner loop parameter
let sum = 0
for i = 1, 4, 1 {
    for j = i, i + 2, 1 {
        sum = sum + i * 10 + j
        print(i * 10 + j)
    }
}
print(sum)
