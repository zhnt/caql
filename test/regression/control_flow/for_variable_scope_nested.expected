11
12
22
23
33
34
135
